browser-client.cpp

```diff
@@ -198,6 +198,30 @@ bool BrowserClient::OnProcessMessageReceived(
 	case ControlLevel::Basic:
 		if (name == "saveReplayBuffer") {
 			obs_frontend_replay_buffer_save();
+		} else if (name == "setBrowserSize") {
+			int width = input_args->GetInt(1);
+			int height = input_args->GetInt(2);
+			if (!width)
+				width = (int)round(input_args->GetDouble(1));
+			if (!height)
+				height = (int)round(input_args->GetDouble(2));
+			if (width > 0 && height > 0) {
+				obs_data_t *s =
+					obs_source_get_settings(bs->source);
+				if (s &&
+				    (obs_data_get_int(s, "width") != width ||
+				     obs_data_get_int(s, "height") != height)) {
+					obs_data_set_int(s, "width", width);
+					obs_data_set_int(s, "height", height);
+					obs_source_update(bs->source, nullptr);
```

Comment:
`bs->source` probably should be `s` just to be safe.

Reply:
I am not sure what you mean as `bs->source` is a `obs_source_t*` and `s` is a `obs_data_t*`.